### ISSUE TITLE
builds can't find asciidoc, even though it's installed

### ISSUE THREAD

**Issue by kived** [Homebrew/homebrew-core#5082]
Trying to build ccache (`brew install --HEAD ccache`) - the application build itself succeeds, but fails to build a doc which fails the entire build.

```
$ asciidoc

Man page:     asciidoc --help manpage
Syntax:       asciidoc --help syntax
$ brew update
Already up-to-date.
$ which asciidoc
/usr/local/bin/asciidoc
$ brew install --HEAD ccache
==> Using the sandbox
==> Cloning https://github.com/ccache/ccache.git
Updating /Users/ryan.pessa/Library/Caches/Homebrew/ccache--git
==> Checking out branch master
==> ./autogen.sh
==> ./configure --prefix=/usr/local/Cellar/ccache/HEAD-31c67cb --mandir=/usr/local/Cellar/ccache/HEAD-31c67cb/share/man --with-bundled-zlib
==> make
==> make install
Last 15 lines from /Users/ryan.pessa/Library/Logs/Homebrew/ccache/04.make:
2016-09-21 15:35:04 -0700

make
install

asciidoc -a revnumber=unknown -d manpage -b docbook MANUAL.txt
make: asciidoc: No such file or directory
make: *** [MANUAL.xml] Error 1

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/Homebrew/homebrew-core/issues
```

https://gist.github.com/anonymous/3304677d71f4933c1ac04013071c62d4

`asciidoc` was not initially installed, so I installed it (`brew install asciidoc`), and I can run it fine from the terminal. But the build still fails.

**Comment by zmwangx** [comment]
The `PATH` when you actually build `ccache --HEAD` is

```
/usr/local/Homebrew/Library/Homebrew/shims/super:/usr/local/opt/autoconf/bin:/usr/local/opt/automake/bin:/usr/local/opt/libtool/bin:/usr/local/opt/git/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

so `asciidoc` is not available.

Fix in #5083.

**Comment by zmwangx** [comment]
@kived Please verify #5083 works for you, thanks.
